src/components/[guild]/AddAndOrderRoles/components/AddRoleDrawer.tsx

```diff
@@ -53,32 +51,8 @@ const AddRoleDrawer = ({ isOpen, onClose, finalFocusRef }): JSX.Element => {
       },
     })
 
-  const defaultValues: RoleToCreate = {
```

Comment:
I think we use both uncontrolled and controlled fields deep inside our form, so let's keep the default values please & just remove the `discordPlatform ? ...` condition ;) (and set an empty array as default value)